Build failure on macOS due to nonexistent field

Trying to build an example you get this error
```sh
error[E0609]: no field `inner` on type `&AppRootInner`
   --> /Users/weethet/.cargo/registry/src/index.crates.io-6f17d22bba15001f/masonry-0.1.2/src/app_root.rs:846:14
    |
846 |         self.inner.borrow().app.hide()
    |              ^^^^^ unknown field
    |
    = note: available fields are: `app_handle`, `debug_logger`, `app_delegate`, `command_queue`, `action_queue` ... and 8 others
```
I suppose this is due to maintainers not having macOS laptop, so it's hard to check, but maybe github actions could bes set up to prevent this any further

--- a/src/app_root.rs
+++ b/src/app_root.rs
@@ -843,13 +843,13 @@ impl AppRootInner {
     #[cfg(target_os = "macos")]
     fn hide_app(&self) {
         use druid_shell::platform::mac::ApplicationExt as _;
-        self.inner.borrow().app.hide()
+        self.app_handle.hide();
     }
 
     #[cfg(target_os = "macos")]
     fn hide_others(&mut self) {
         use druid_shell::platform::mac::ApplicationExt as _;
-        self.inner.borrow().app.hide_others();
+        self.app_handle.hide_others();
     }
 }
 

